.devcontainer/development/local-features/conda-packages/install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Install conda packages needed for Nextflow Training
+
+conda config --add channels bioconda
+conda config --add channels conda-forge
+conda config --set channel_priority strict
+conda update --quiet --yes --all
+
+conda install --quiet --yes --name base \
+    mamba \
+    nextflow \
+    nf-core \
+    nf-test \
+    black \
+    prettier \
+    pre-commit \
```

Comment:
What do you think about installing python tooling via uv tool install like I do in #568 's postCreateCommand.sh

Reply:
The postCreateCommand was just to point you to the right part of the code. It only takes a fraction of a second to complete, so it seemed unnecessary to include it in the build. I'm good with it in either place.

We can save the pixi implementation for a later improvement.